dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem_themeresources.xaml

```diff
@@ -2,66 +2,68 @@
 <ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
 
     <Style TargetType="local:RadioMenuFlyoutItem" BasedOn="{StaticResource DefaultRadioMenuFlyoutItemStyle}" />
 
-    <!-- Same as default rs1 style for ToggleMenuFlyoutItem -->
+    <!-- Same as DefaultToggleMenuFlyoutItemStyle -->
     <Style x:Key="DefaultRadioMenuFlyoutItemStyle" TargetType="local:RadioMenuFlyoutItem">
-        <Setter Property="Background" Value="{ThemeResource ToggleMenuFlyoutItemBackground}" />
-        <Setter Property="Foreground" Value="{ThemeResource ToggleMenuFlyoutItemForeground}" />
-        <Setter Property="Padding" Value="{ThemeResource MenuFlyoutItemThemePadding}" />
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Foreground" Value="{ThemeResource MenuFlyoutItemForeground}" />
```

Comment:
Perhaps you should create separate resources for RadioMenuFlyoutItem specifically, to give users the freedom to update its colours without affecting MenuFlyoutItem?

Reply:
I can't quite imagine when someone might want to have it differ from MenuFlyoutItem, but sometimes I am surprised. :P (I do think the "Transparent" background should probably be MenuFlyoutItemBackground, though)

Reply:
@YuliKl @ranjeshj can you weight in on this? should we create separate resources?

Reply:
Ah true, the `Background` should be `MenuFlyoutItemBackground` pointing to Transparent. Other than that, I'm ok not adding new resources for now (since it's easier to add than remove? 😂).